internal/utils/kubeutils.go

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) 2020 Doc.ai and/or its affiliates.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package utils
+
+import (
+	"os"
+	"path/filepath"
+
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+// NewClientSetConfig creates ClientSetConfig from config file.
+// If config file path not provided via env variable KUBECONFIG, default path "HOME/.kube/config" will be used
+func NewClientSetConfig() (*rest.Config, error) {
```

Comment:
@denis-tingajkin should we consider moving all this stuff into sdk-k8s?

Reply:
I think it is a good idea.